When I use pojo as the RequestParam, NPE will be thrown.

### Expected behavior
work normal.

### Actual behavior
throw NPE
```
io.esastack.restlight.server.bootstrap.WebServerException: java.lang.NullPointerException
	at io.esastack.restlight.server.bootstrap.WebServerException.wrap(WebServerException.java:61) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.core.handler.impl.AbstractExecution.resolveArgs(AbstractExecution.java:106) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.core.handler.impl.AbstractRouteExecution.lambda$doHandle$0(AbstractRouteExecution.java:92) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at java.base/java.util.concurrent.CompletableFuture.uniApplyNow(CompletableFuture.java:680) ~[na:na]
	at java.base/java.util.concurrent.CompletableFuture.uniApplyStage(CompletableFuture.java:658) ~[na:na]
	at java.base/java.util.concurrent.CompletableFuture.thenApply(CompletableFuture.java:2094) ~[na:na]
	at java.base/java.util.concurrent.CompletableFuture.thenApply(CompletableFuture.java:143) ~[na:na]
	at io.esastack.restlight.core.handler.impl.AbstractRouteExecution.doHandle(AbstractRouteExecution.java:87) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.core.handler.LinkedRouteFilterChain.lambda$immutable$0(LinkedRouteFilterChain.java:48) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.core.spi.impl.RouteTracking.routed(RouteTracking.java:55) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.core.handler.LinkedRouteFilterChain.doNext(LinkedRouteFilterChain.java:60) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.core.handler.impl.AbstractRouteExecution.handle(AbstractRouteExecution.java:78) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.server.bootstrap.DispatcherHandlerImpl.service(DispatcherHandlerImpl.java:82) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.server.schedule.ScheduledHandler.lambda$null$1(ScheduledHandler.java:83) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:859) ~[na:na]
	at java.base/java.util.concurrent.CompletableFuture.uniWhenCompleteStage(CompletableFuture.java:883) ~[na:na]
	at java.base/java.util.concurrent.CompletableFuture.whenComplete(CompletableFuture.java:2251) ~[na:na]
	at io.esastack.restlight.server.schedule.ScheduledHandler.lambda$processByFixedScheduler$2(ScheduledHandler.java:74) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.server.schedule.RequestTaskImpl.run(RequestTaskImpl.java:45) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at esa.commons.concurrent.DirectExecutor.execute(DirectExecutor.java:32) ~[commons-0.2.1-20220331.024855-1.jar:na]
	at io.esastack.restlight.server.schedule.ExecutorSchedulerImpl.schedule(ExecutorSchedulerImpl.java:38) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.server.schedule.ScheduledHandler.processByFixedScheduler(ScheduledHandler.java:88) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.server.schedule.ScheduledHandler.lambda$new$0(ScheduledHandler.java:48) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.server.schedule.ScheduledHandler.process(ScheduledHandler.java:64) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.server.schedule.ScheduledRestlightHandler.process(ScheduledRestlightHandler.java:116) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.server.schedule.ScheduledRestlightHandler.process(ScheduledRestlightHandler.java:45) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.server.schedule.AbstractRestlightHandler.process(AbstractRestlightHandler.java:46) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.core.handler.RestlightHandlerImpl.process(RestlightHandlerImpl.java:75) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.server.bootstrap.NettyRestlightServer.lambda$null$5(NettyRestlightServer.java:254) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.httpserver.impl.BaseRequestHandle.handleEnd(BaseRequestHandle.java:272) ~[httpserver-0.1.1-20211213.125310-5.jar:na]
	at io.esastack.httpserver.impl.Http1Handler.end(Http1Handler.java:318) ~[httpserver-0.1.1-20211213.125310-5.jar:na]
	at io.esastack.httpserver.impl.Http1Handler.channelRead0(Http1Handler.java:149) ~[httpserver-0.1.1-20211213.125310-5.jar:na]
	at io.esastack.httpserver.impl.Http1Handler.channelRead0(Http1Handler.java:57) ~[httpserver-0.1.1-20211213.125310-5.jar:na]
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) ~[netty-codec-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) ~[netty-codec-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286) ~[netty-handler-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324) ~[netty-codec-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296) ~[netty-codec-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:714) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:650) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:576) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493) ~[netty-transport-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) ~[netty-common-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.52.Final.jar:4.1.52.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.52.Final.jar:4.1.52.Final]
	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
Caused by: java.lang.NullPointerException: null
	at io.esastack.restlight.core.spi.impl.DefaultStringConverterFactory$1.fromString(DefaultStringConverterFactory.java:43) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.core.resolver.nav.NameAndStringsValueResolver.resolve(NameAndStringsValueResolver.java:74) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.core.resolver.nav.NameAndValueResolverAdapter.resolve(NameAndValueResolverAdapter.java:42) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.core.handler.impl.AdvisedParamResolver.resolve(AdvisedParamResolver.java:41) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	at io.esastack.restlight.core.handler.impl.AbstractExecution.resolveArgs(AbstractExecution.java:103) ~[restlight-core-1.0.0-SNAPSHOT.jar:na]
	... 65 common frames omitted
```

### Steps to reproduce
1.use the @RequestParam to declare a parameter.
2.request the uri.

### Env
- Restlight version: 1.0.0-SNAPSHOT
- JVM version: 1.8
- OS:windows 10

--- a/restlight-core/src/main/java/io/esastack/restlight/core/resolver/param/RequestBeanParamResolver.java
+++ b/restlight-core/src/main/java/io/esastack/restlight/core/resolver/param/RequestBeanParamResolver.java
@@ -48,7 +48,7 @@ import java.util.stream.Collectors;
  */
 public class RequestBeanParamResolver implements ParamResolverFactory {
 
-    private static final Map<Class<?>, ParamResolver> META_CACHE = new ConcurrentHashMap<>(16);
+    protected final Map<Class<?>, ParamResolver> metaCache = new ConcurrentHashMap<>(16);
     private final DeployContext ctx;
 
     public RequestBeanParamResolver(DeployContext ctx) {
@@ -70,10 +70,10 @@ public class RequestBeanParamResolver implements ParamResolverFactory {
         Class<?> type = param.type();
         // instantiate target object by unsafe
 
-        ParamResolver resolver = META_CACHE.get(type);
+        ParamResolver resolver = metaCache.get(type);
         if (resolver == null) {
             // no need to check the previous value
-            META_CACHE.putIfAbsent(type, resolver = new Resolver(newTypeMeta(type,
+            metaCache.putIfAbsent(type, resolver = new Resolver(newTypeMeta(type,
                     converters,
                     ctx.resolverFactory().orElse(null))));
         }

--- a/restlight-core/src/main/java/io/esastack/restlight/core/spi/impl/DefaultStringConverterFactory.java
+++ b/restlight-core/src/main/java/io/esastack/restlight/core/spi/impl/DefaultStringConverterFactory.java
@@ -36,15 +36,16 @@ public class DefaultStringConverterFactory implements StringConverterFactory {
         } else {
             converter = ConverterUtils.str2ObjectConverter(key.type());
         }
-        if (converter == null) {
-            throw new IllegalArgumentException("There is no suitable StringConverter for class(" + key.type() + "),"
-                    + "It should have a constructor that accepts a single String argument or "
-                    + "have a static method named valueOf() or fromString() that accepts a single String argument.");
-        }
 
         return Optional.of(new StringConverter() {
             @Override
             public Object fromString(String value) {
+                if (converter == null) {
+                    throw new IllegalArgumentException("There is no suitable StringConverter for class(" + key.type() + "),"
+                            + "It should have a constructor that accepts a single String argument or "
+                            + "have a static method named valueOf() or fromString() that accepts a single String argument.");
+                }
+
                 return converter.apply(value);
             }
 

--- a/restlight-core/src/main/java/io/esastack/restlight/core/spi/impl/DefaultStringConverterFactory.java
+++ b/restlight-core/src/main/java/io/esastack/restlight/core/spi/impl/DefaultStringConverterFactory.java
@@ -41,9 +41,10 @@ public class DefaultStringConverterFactory implements StringConverterFactory {
             @Override
             public Object fromString(String value) {
                 if (converter == null) {
-                    throw new IllegalArgumentException("There is no suitable StringConverter for class(" + key.type() + "),"
-                            + "It should have a constructor that accepts a single String argument or "
-                            + "have a static method named valueOf() or fromString() that accepts a single String argument.");
+                    throw new IllegalArgumentException("There is no suitable StringConverter for class(" +
+                            key.type() + "),It should have a constructor that accepts a single String argument" +
+                            " or have a static method named valueOf() or fromString() that accepts a " +
+                            "single String argument.");
                 }
 
                 return converter.apply(value);

--- a/restlight-integration-test/springmvc-test/src/main/java/io/esastack/restlight/integration/springmvc/cases/annotation/CustomFieldParam.java
+++ b/restlight-integration-test/springmvc-test/src/main/java/io/esastack/restlight/integration/springmvc/cases/annotation/CustomFieldParam.java
@@ -1,0 +1,26 @@
+/*
+ * Copyright 2022 OPPO ESA Stack Project
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.esastack.restlight.integration.springmvc.cases.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface CustomFieldParam {
+}

--- a/restlight-integration-test/springmvc-test/src/main/java/io/esastack/restlight/integration/springmvc/cases/config/ResolverConfig.java
+++ b/restlight-integration-test/springmvc-test/src/main/java/io/esastack/restlight/integration/springmvc/cases/config/ResolverConfig.java
@@ -29,11 +29,12 @@ import io.esastack.restlight.core.resolver.StringConverterProvider;
 import io.esastack.restlight.core.resolver.rspentity.AbstractResponseEntityResolver;
 import io.esastack.restlight.core.serialize.HttpRequestSerializer;
 import io.esastack.restlight.core.serialize.HttpResponseSerializer;
+import io.esastack.restlight.integration.springmvc.cases.annotation.CustomFieldParam;
 import io.esastack.restlight.integration.springmvc.cases.annotation.CustomRequestBean;
-import io.esastack.restlight.server.context.RequestContext;
 import io.esastack.restlight.integration.springmvc.cases.annotation.CustomRequestBody;
 import io.esastack.restlight.integration.springmvc.cases.annotation.CustomResponseBody;
 import io.esastack.restlight.integration.springmvc.entity.UserData;
+import io.esastack.restlight.server.context.RequestContext;
 import io.netty.handler.codec.http.HttpHeaderNames;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
@@ -123,6 +124,22 @@ public class ResolverConfig {
             @Override
             public boolean supports(HandlerMethod method) {
                 return method.hasMethodAnnotation(CustomResponseBody.class, true);
+            }
+        };
+    }
+
+    @Bean
+    public ParamResolverFactory customParamResolverFactory() {
+        return new ParamResolverFactory() {
+            @Override
+            public ParamResolver createResolver(Param param, StringConverterProvider converters,
+                                                List<? extends HttpRequestSerializer> serializers) {
+                return context -> context.request().getParam("name");
+            }
+
+            @Override
+            public boolean supports(Param param) {
+                return param.hasAnnotation(CustomFieldParam.class);
             }
         };
     }

--- a/restlight-integration-test/springmvc-test/src/main/java/io/esastack/restlight/integration/springmvc/cases/controller/AnnotationController.java
+++ b/restlight-integration-test/springmvc-test/src/main/java/io/esastack/restlight/integration/springmvc/cases/controller/AnnotationController.java
@@ -111,4 +111,9 @@ public class AnnotationController {
     public String customResponseBody(@RequestParam String name) {
         return name;
     }
+
+    @GetMapping("get/param/wrong")
+    public UserData paramWrong(@RequestParam UserData user) {
+        return user;
+    }
 }

--- a/restlight-integration-test/springmvc-test/src/main/java/io/esastack/restlight/integration/springmvc/entity/UserData.java
+++ b/restlight-integration-test/springmvc-test/src/main/java/io/esastack/restlight/integration/springmvc/entity/UserData.java
@@ -13,11 +13,14 @@
 
 package io.esastack.restlight.integration.springmvc.entity;
 
+import io.esastack.restlight.integration.springmvc.cases.annotation.CustomFieldParam;
+
 import java.math.BigDecimal;
 import java.util.Date;
 
 public class UserData {
 
+    @CustomFieldParam
     private String name;
 
     private Integer age;

--- a/restlight-integration-test/springmvc-test/src/main/java/io/esastack/restlight/integration/springmvc/test/AnnotationTest.java
+++ b/restlight-integration-test/springmvc-test/src/main/java/io/esastack/restlight/integration/springmvc/test/AnnotationTest.java
@@ -138,4 +138,11 @@ public class AnnotationTest extends BaseIntegrationTest {
         UserData user = response.bodyToEntity(UserData.class);
         Assert.assertEquals("test", user.getName());
     }
+
+    @Test
+    public void testParamWrong() throws Exception {
+        RestResponseBase response = restClient.get(domain + "/annotation/get/param/wrong")
+                .addParam("user", "").execute().toCompletableFuture().get();
+        Assert.assertEquals(HttpStatus.INTERNAL_SERVER_ERROR.code(), response.status());
+    }
 }

--- a/restlight-integration-test/springmvc-test/src/main/java/io/esastack/restlight/integration/springmvc/test/AwareTest.java
+++ b/restlight-integration-test/springmvc-test/src/main/java/io/esastack/restlight/integration/springmvc/test/AwareTest.java
@@ -14,7 +14,6 @@
 package io.esastack.restlight.integration.springmvc.test;
 
 import io.esastack.restclient.RestResponseBase;
-import io.netty.channel.nio.NioEventLoopGroup;
 import org.junit.Assert;
 import org.junit.Test;
 
@@ -33,7 +32,7 @@ public class AwareTest extends BaseIntegrationTest {
     public void testIoAware() throws Exception {
         RestResponseBase response = restClient.get(domain + "/aware/get/io").execute()
                 .toCompletableFuture().get();
-        Assert.assertEquals(NioEventLoopGroup.class.getName(), response.bodyToEntity(String.class));
+        Assert.assertTrue(response.bodyToEntity(String.class).contains("EventLoopGroup"));
     }
 
     @Test

--- a/restlight-jaxrs-provider/src/test/java/io/esastack/restlight/jaxrs/adapter/MessageBodyReaderAdapterTest.java
+++ b/restlight-jaxrs-provider/src/test/java/io/esastack/restlight/jaxrs/adapter/MessageBodyReaderAdapterTest.java
@@ -48,7 +48,7 @@ class MessageBodyReaderAdapterTest {
     void testBasic() {
         assertThrows(NullPointerException.class, () -> new MessageBodyWriterAdapter<>(null));
         MessageBodyReaderAdapter<?> adapter = new MessageBodyReaderAdapter<>(mock(Providers.class));
-        assertEquals(Ordered.LOWEST_PRECEDENCE, adapter.getOrder());
+        assertEquals(90, adapter.getOrder());
         assertTrue(adapter.supports(mock(Param.class)));
     }
 

--- a/restlight-jaxrs-provider/src/test/java/io/esastack/restlight/jaxrs/adapter/MessageBodyWriterAdapterTest.java
+++ b/restlight-jaxrs-provider/src/test/java/io/esastack/restlight/jaxrs/adapter/MessageBodyWriterAdapterTest.java
@@ -65,7 +65,7 @@ class MessageBodyWriterAdapterTest {
 
         final Providers providers = mock(Providers.class);
         final MessageBodyWriterAdapter<?> adapter = new MessageBodyWriterAdapter<>(providers);
-        assertEquals(Ordered.LOWEST_PRECEDENCE, adapter.getOrder());
+        assertEquals(90, adapter.getOrder());
 
         final HttpResponse response = mock(HttpResponse.class);
         final ResponseEntity entity = mock(ResponseEntity.class);

